docs/traffic-policy/actions/log/config.mdx

```diff
@@ -0,0 +1,19 @@
+## Configuration Reference
+
+This is the [Traffic Policy](/docs/http/traffic-policy/index.mdx) configuration 
+reference for this action.
+
+### Action Type
+
+`log`
+
+### Configuration Fields
+
+| Parameter  | Type                   | Description                                                                            |
+| ---------- | ---------------------- | -------------------------------------------------------------------------------------- |
+| `metadata` | `map[string]any`       | **Required.** A key-value map of metadata that you would like to include in your events for this action. Supports [string interpolation](#string-interpolation). |
+
+### Supported Directions
+
+- `inbound`
+- `outbound`
```

Comment:
maybe double check if outbound is supported for tcp/tls, i think today it's not, but in future?

Reply:
`	DirectionsSupported: []pe.Direction{pe.Inbound, pe.Outbound},`

It's there, never tried it, but I would assume it works. /shrug